Core/Object Arts/Dolphin/ActiveX/OCX/AXControlSite.cls

```diff
@@ -795,6 +795,29 @@ state
 	sequence list addAll: controlState afterIndex: 1.
 	^sequence!
 
+storeOn: aStream indent: anInteger variables: aDictionary
+
+	| default name |
+	name := self storeNameIn: aDictionary.
+	aStream display: name; cr.
+	anInteger + 1 timesRepeat: [aStream tab].
+	[
+		default := (self class progId: self progId) create; show.
+		aStream
+			nextPutAll: ' progId: ';
```

Comment:
How far down the rabbit hole do I go with this? #new? #yourself? How do I handle keyword selectors with multiple arguments?